internal/guest/runtime/hcsv2/container.go

```diff
@@ -183,20 +186,32 @@ func (c *Container) setExitType(signal syscall.Signal) {
 }
 
 // GetStats returns the cgroup metrics for the container.
-func (c *Container) GetStats(ctx context.Context) (*v1.Metrics, error) {
+func (c *Container) GetStats(ctx context.Context) (_ *v1.Metrics, err error) {
 	_, span := trace.StartSpan(ctx, "opengcs::Container::GetStats")
 	defer span.End()
-	span.AddAttributes(trace.StringAttribute("cid", c.id))
+	defer func() { oc.SetSpanStatus(span, err) }()
+	span.AddAttributes(trace.StringAttribute(logfields.ContainerID, c.id))
 
-	cgroupPath := c.spec.Linux.CgroupsPath
-	cg, err := cgroups.Load(cgroups.V1, cgroups.StaticPath(cgroupPath))
+	cg, err := c.GetCGroup(ctx)
 	if err != nil {
 		return nil, errors.Errorf("failed to get container stats for %v: %v", c.id, err)
 	}
 
 	return cg.Stat(cgroups.IgnoreNotExist)
 }
 
+// GetCGroup returns the container's cgroup
+func (c *Container) GetCGroup(ctx context.Context) (_ cgroups.Cgroup, err error) {
```

Comment:
`Cgroup` seems to be the preferred capitalization, so suggest renaming to `GetCgroup` or just `Cgroup`.